src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,22 +25,34 @@
   private String user;
   private String text;
   private long timestamp;
-
+  private String recipient; 
   /**
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
    * random ID and uses the current system time for the creation time.
    */
   public Message(String user, String text) {
```

Comment:
This looks good but I think you want to remove these two constructors instead of adding new ones.

Reply:
so am i removing both the constructors ?

Reply:
Well i think what you want to do ultimately is add the `String recipient` parameter to the two constructors that are already in this class instead of creating two new constructors.